core/src/main/java/bio/terra/pearl/core/service/workflow/RegistrationService.java

```diff
@@ -125,6 +137,33 @@ private RegistrationResult register(String portalShortcode, EnvironmentName envi
         return new RegistrationResult(user, ppUser);
     }
 
+    public RegistrationResult registerGovernedUser(String portalShortcode, ParticipantUser proxy) {
```

Comment:
this method should have @Transactional on it for the case where it's called on it's own (e.g. if a proxy is registering a second governed user, and so this isn't being called from within another registration transaction